src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
```

Comment:
should all animals have `hidden` method?

Reply:
If we are talking about animals. Carnivores hide no worse than herbivores))